### ISSUE TITLE
node / edge export characters in sscandinavian upper ascii

### ISSUE THREAD

**Issue by mihkal** [gephi/gephi#1821]
## Expected Behavior
to export correctly upper ascii chars

## Current Behavior
![image](https://user-images.githubusercontent.com/23520032/31653936-fc4626c8-b32c-11e7-8a09-2939691269aa.png)

not currently doing so

## Steps to Reproduce

1.
2.
3.
4.

## Your Environment

* Version used: Gephi 0.9.2
* Java version: 
* Operating System:

**Comment by eduramiba** [comment]
Are you using spreadsheet (CSV) export? Make sure to select UTF-8 encoding in the export settings, and also make sure that your text editor or spreadsheet software detects correctly that the file encoding is UTF-8 (this usually works weird in excel). This might help https://superuser.com/questions/280603/how-to-set-character-encoding-when-opening-excel

**Comment by mihkal** [comment]
UTF-8 
hmmm, I may be blind but can not see that option anywhere

**Comment by eduramiba** [comment]
During export or opening the file? Right, spreadsheet import options no longer includes encoding option, it always uses UTF-8 by default.

What kind of export do you use? What program do you use to open the file?

**Comment by eduramiba** [comment]
Was it solved?

**Comment by eduramiba** [comment]
I assume it worked?

**Comment by mihkal** [comment]
No it did not work. there is no encoding option in export

to 26. lokak. 2017 klo 0.04 Eduardo Ramos <notifications@github.com>
kirjoitti:

**Comment by eduramiba** [comment]
It's always exported un UTF-8 as all file exporters in Gephi. It's in your other tool where you need to specify that

**Comment by mihkal** [comment]
and your export uses , as delimiter not the ; as in our locale.
this means that csv files never open correctly in our parts of the world.

to 26. lokak. 2017 klo 9.50 Eduardo Ramos <notifications@github.com>
kirjoitti:

**Comment by eduramiba** [comment]
Right, it uses RFC 4180 by default but you still can change that in export settings.

What I am trying to say is Excel is not the only CSV tool, and it uses its own quircky locale dependent settings to open and write CSV, and hiding this behavior to the user by default. You can make It open a CSV with any configuration with some advanced menu.

Other tools such as Libre Office handle this better because they guess the CSV parsing settings, they show them before import and they allow the user to manually change them (so you can open files coming from any locale painlessly, like in gephi).
